packages/@react-spectrum/view/src/SVGIllustration.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, filterDOMProps, useSlotProps, useStyleProps} from '@react-spectrum/utils';
+import {DOMProps, SVGProps, ViewStyleProps} from '@react-types/shared';
+import {HTMLElement} from 'react-dom';
+import React, {ReactElement, ReactSVGElement, RefObject} from 'react';
+
+export interface IllustrationProps extends DOMProps, SVGProps, ViewStyleProps {
+  children: ReactElement | ReactElement[]
+}
+
+export const SVGIllustration = React.forwardRef((props: IllustrationProps, ref: RefObject<HTMLElement>) => {
+  props = useSlotProps(props, 'illustration');
+  console.log('props', props);
+  let {
+    children,
+    ...otherProps
+  } = props;
+  let {styleProps} = useStyleProps(otherProps);
+
+  return (
+    <svg {...filterDOMProps(otherProps)} {...styleProps} className={classNames({}, styleProps.className)} ref={ref}>
```

Comment:
```packages/@react-spectrum/view/src/SVGIllustration.tsx:32:6 - error TS2322: Type '{ children: ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)>) | (new (props: any) => Component<any, any, any>)> | ReactElement<...>[]; ... 253 more ...; css?: InterpolationWithTheme<...>; }' is not assignable to type 'SVGProps<SVGSVGElement>'.
  Types of property 'onCopy' are incompatible.
    Type '(event: ClipboardEvent<HTMLElement>) => void' is not assignable to type '(event: ClipboardEvent<SVGSVGElement>) => void'.
      Types of parameters 'event' and 'event' are incompatible.
        Type 'ClipboardEvent<SVGSVGElement>' is not assignable to type 'ClipboardEvent<HTMLElement>'.
          Type 'SVGSVGElement' is missing the following properties from type 'HTMLElement': accessKey, accessKeyLabel, autocapitalize, dir, and 16 more.

32     <svg {...filterDOMProps(otherProps)} {...styleProps} className={classNames({}, styleProps.className)} ref={ref}>
        ~~~
```

Reply:
I think you won't have this issue if you do what Icon and UIIcon do